docs/roman/dark_current/description.rst

```diff
@@ -0,0 +1,57 @@
+Description
+===========
+
+Assumptions
+-----------
+
+It is assumed that the input science data have had the zero group (or
+bias) subtracted. Accordingly, the dark reference data
+should have their own group zero subtracted from all groups. 
+
+Algorithm
+---------
+
+The dark current step removes dark current from a Roman exposure by subtracting
+dark current data stored in a dark reference file.
+
+The current implementation uses dark reference files that have been
+constructed from exposures using NFRAMES=1 and GROUPGAP=0 (i.e. one
+frame per group and no dropped frames) and the maximum number of frames
+allowed for an integration. If the science exposure that's being processed
+also used NFRAMES=1 and GROUPGAP=0, then the dark reference file data
+are directly subtracted frame-by-frame from the science exposure.
+
```

Comment:
I'm not sure this is quite right for Roman. The WFI MA table will combine NFRAMES to make a resultant. Each WFI exposure will contain some # of resultants. GROUPGAP is nskips?

As I understand it right now after the discussions with Roman and INS is that for each MA table we will have a reference dark. So selecting a dark should just use the metadata for the exposure, which specifies the MA table, and the appropriate dark is returned. So we will not be constructing a dark reference file. This may change but for now this is what I think will happen.
So the dark reference is subtracted based on exposure and each resultant has the dark current subtracted, what JWST calls group-by-group.
Yes having JWST code and nomenclature and mixing it with the Roman vocabulary is simply confusing.

Reply:
I have modified the text to reflect this. Please let me know if it needs further tweaking.